requirements.txt

```diff
@@ -1,3 +1,4 @@
+numpy
```

Comment:
This is only used at https://github.com/n1analytics/anonlink/blob/develop/anonlink/network_flow.py#L138. The use in `network_flow.py` should be removed as it's a single trivial use for such a huge dependency.